Projects/hCoreUI/Sources/hForm/hFloatingField.swift

```diff
@@ -10,6 +10,7 @@ public struct hFloatingField: View {
     @Binding var error: String?
     private var value: String
     private let onTap: () -> Void
+    private let lockedState: Bool
```

Comment:
we can remove this here and use 
environment what we have
`    @Environment(\.hFieldLockedState) var isLocked
`

if we add additional extension here
```
extension View {
    public var hFieldLockedState: some View {
        self.environment(\.hFieldLockedState, true)
    }
}
```

with 

```
extension View {
    public var hFieldLockedState: some View {
        self.environment(\.hFieldLockedState, true)
    }
    
    public func hFieldSetLockedState(to value: Bool) -> some View {
        self.environment(\.hFieldLockedState, value)
    }
}

```